### PR TITLE
[1LP][RFR]Implement test_add_second_provider

### PR DESCRIPTION
**Pull request by mmojzis**
{{ pytest: -k "test_add_second_provider" -v }}

### CODE REVIEW

File: cfme/tests/cloud/test_providers.py [hunk 1]
```diff
@@ -1397,3 +1398,30 @@ def test_add_ec2_provider_with_sts_assume_role(appliance, ec2_provider_with_sts_
...
+            2. Add EC2 Provider
+            3. Add another EC2 Provider
+        expectedResults:
+            1.
+            2. Provider should be successfully added.
+            3. Provider should be successfully added.
+    """
+    second_provider = provider
+    second_provider.name = "{}-2".format(provider.name)
+    second_provider.create()
```
[the remark this answers is not in the document]
Comment (mmojzis) [reply cfme/tests/cloud/test_providers.py]: Okay, that makes sense. Added

File: cfme/tests/cloud/test_providers.py [hunk 2]
```diff
@@ -1397,3 +1398,34 @@ def test_add_ec2_provider_with_sts_assume_role(appliance, ec2_provider_with_sts_
...
+        testSteps:
+            1. Go to Compute -> Cloud -> Providers
+            2. Add EC2 Provider
+            3. Add another EC2 Provider
+        expectedResults:
+            1.
+            2. Provider should be successfully added.
+            3. Provider should be successfully added.
+    """
+    second_provider = provider
```
Comment (mshriver) [inline cfme/tests/cloud/test_providers.py]: This cannot be done via simple assignment, as it modifies the original instance.

```
In [2]: provider                                                                                                              
Out[2]: EC2Provider(name='ec2-west', key='ec2west', zone='default', region='us-west-1', region_name='US West (N. California)')

In [3]: second_provider=provider                                                                                              

In [4]: second_provider.name='junk name'                                                                                      

In [5]: provider                                                                                                              
Out[5]: EC2Provider(name='junk name', key='ec2west', zone='default', region='us-west-1', region_name='US West (N. California)')

```

It has to be done using `deepcopy`

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

File: cfme/tests/cloud/test_providers.py (hunk 2 above)
Comment (mshriver) [reply cfme/tests/cloud/test_providers.py]: For those following along at home - deepcopy actually can't be used with the provider instances, but the `cfme.utils.providers.get_crud` method can be used to fetch another provider instance using the key.

Comment (mmojzis) [reply cfme/tests/cloud/test_providers.py]: Reworked with using get_crud.

**Pull request merged**